gui/storage/forms.py

```diff
@@ -1765,7 +1765,10 @@ def __init__(self, *args, **kwargs):
         volnames = [
             o.vol_name for o in models.Volume.objects.filter(vol_fstype='ZFS')
         ]
```

Comment:
getting volnames on my interpreter fails since there is no vol_fstype anymore, didn't this fail for you too?